src/lwc/healthCheckRun/healthCheckRun.html

```diff
@@ -1,32 +1,5 @@
 <template>
     <lightning-card title={labelReference.stgHealthCheckTitle}>
-        <h3 slot="title">
-            <div class="slds-grid">
-                <div class="slds-col">
-                    <lightning-icon
-                        icon-name="standard:home"
-                        size="medium"
-                    ></lightning-icon>
-                </div>
-                <div class="slds-col 
-                            slds-grid 
-                            slds-wrap 
-                            slds-text-align_left 
-                            slds-p-horizontal_small">
-                    <div class="slds-col 
-                                slds-size_1-of-1 
-                                slds-text-body_regular">
-                        {labelReference.stgEDASetupTitle}
-                    </div>
-                    <div class="slds-col 
-                                slds-size_1-of-1 
-                                slds-text-title">
-                        {labelReference.stgHealthCheckTitle}
-                    </div>
-                </div>
-            </div>
-        </h3>
-
```

Comment:
Please do not remove this for now. We are commenting it out. See the PR for tab opening.

Reply:
This seems to be code that was duplicated from the healthCheckHighlightsPanel.html markup (which I may have neglected to remove after copy/past).  I'm not sure it belongs in the healthCheckRun component?

Reply:
Oh, very well. Carry on then!